### ISSUE TITLE
Add non-Docker instructions for local testing prior to PR submission.

### ISSUE THREAD

**Issue by ritjoe** [FOSSRIT/fossrit.github.io#80]
These would go in .github/CONTRIBUTING.md

I believe @jrtechs mentioned maybe being able to contribute something here? 

Might do to see if something has been added along these lines to https://github.com/RITlug/ritlug.github.io

This is complementary to, but in a way in the opposite direction from, #36

**Comment by ritjoe** [comment]
relevant https://github.com/RITlug/ritlug.github.io/blob/master/CONTRIBUTING.md

**Comment by jrtechs** [comment]
This would be nice to include. Although the docker script is convenient, it is a bit slow at times. Although we do have some documentation on how to do it on the RITlug website, your luck may vary based on your distro. I don't remember whether or not we updated it last lear. I can check it out tonight.

**Comment by jwflory** [comment]
@ritjoe Could you please provide justification for why Docker is not viable for testing locally?

There is limited bandwidth for offering multiple pathways of support for running this application locally. The Docker use case is more attractive to me as a maintainer because I can spend less time debugging with someone when a local environment issue is at play. If I am going to spend time improving the contributing guidelines and the dev environment instructions, I want to spend that time on the Docker use case.

**Comment by ritjoe** [comment]
@jwflory 

I think it would be inappropriate and unhelpful to turn this issue into a debate about the relative merits of our respective tooling preferences.

While contributions are welcome, they aren't expected. If one doesn't have something to offer this particular issue, that's more than fine!

I've assigned myself the issue in hopes that makes clearer who this is for.

When opening the issue I looked for a **wishlist** label to apply to this, but didn't see one.

Is there any objection or concern to creating such a label and applying it to this?
